apigee_sync.go

```diff
@@ -160,6 +161,13 @@ func pollChangeAgent() error {
 			return err
 		}
 
+		if changesRequireDDLSync(resp) {
+			log.Info("Detected DDL changes, going to fetch a new snapshot to sync...")
+			return changeServerError{
```

Comment:
The error is checked by a calling function.  

I guess I see the two cases "snapshot too old" and "new table encountered" to essentially be equivalent (because "new table encountered necessarily means "snapshot too old").  So if one is an error, I feel like both should be, or neither.  But ultimately I don't really care so I'll punt up the chain